packages/terra-divider/src/Divider.jsx

```diff
@@ -1,16 +1,37 @@
 import React from 'react';
 import classNames from 'classnames/bind';
+import PropTypes from 'prop-types';
 import styles from './Divider.module.scss';
 
+const propTypes = {
+  /**
+  * String to be displayed in-line with the divider.
```

Comment:
Should `inline` be one word instead of hyphenated?

Reply:
Maybe.
.
.
.
.
.
.
.
Yes.

Reply:
Changes added! [7ad1cc8](https://github.com/cerner/terra-core/pull/2296/commits/7ad1cc8ca8af947a4b40c5cbf02c9a0bf49c534f)